db/migrations/002_create_table_user_order.up.sql

```diff
@@ -0,0 +1,11 @@
+drop type if exists order_status;
+create type order_status as enum ('NEW', 'PROCESSING', 'INVALID', 'PROCESSED');
+
+drop table if exists user_order;
+create table user_order (
+                            user_id int references usr(id),
+                            number varchar(255) unique not null,
```

Comment:
```suggestion
                            order_nr varchar(255) unique not null,
```
number зарезервированное имя и может не работать в некоторых версиях бд